app/src/components/Modals/add-user-modal.tsx

```diff
@@ -0,0 +1,106 @@
+"use client";
+
+import { ProfileInputs } from "@/app/[lng]/settings/page";
+import {
+  Modal,
+  Button,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+  ModalProps,
+  Input,
+  FormControl,
+  FormLabel,
+  Box,
+} from "@chakra-ui/react";
+import React, { FC } from "react";
+import { useForm } from "react-hook-form";
+import FormInput from "../form-input";
+import FormSelectInput from "../form-select-input";
+
+interface AddUserModalProps {
+  isOpen: boolean;
+  onClose: any;
+}
+
+const AddUserModal: FC<AddUserModalProps> = ({ isOpen, onClose }) => {
+  const {
+    handleSubmit,
+    register,
+    formState: { errors, isSubmitting },
+    setValue,
+  } = useForm<ProfileInputs>();
+  return (
+    <>
+      <Modal blockScrollOnMount={false} isOpen={isOpen} onClose={onClose}>
+        <ModalOverlay />
+        <ModalContent minH="524px" minW="568px" marginTop="10%">
+          <ModalHeader
+            display="flex"
+            justifyContent="center"
+            fontWeight="semibold"
+            fontSize="headline.sm"
+            lineHeight="32"
+            padding="24px"
+            borderBottom={"1px solid #fafafa"}
+          >
+            Add User
+          </ModalHeader>
+          <ModalCloseButton marginTop="10px" />
+          <ModalBody>
+            <form
+              onSubmit={handleSubmit(() => {
+                alert("hello");
+              })}
+            >
+              <Box display="flex" flexDirection="column" gap="24px">
+                <FormInput
+                  id="name"
+                  error={errors.name}
+                  label="Full Name"
+                  register={register}
+                />
+                <FormInput
+                  id="email"
+                  error={errors.email}
+                  label="Email"
+                  register={register}
+                />
+
+                <FormSelectInput
+                  label="Role"
+                  value="admin"
+                  register={register}
+                  error={errors.role}
+                  id="role"
+                  onInputChange={() => {}}
+                />
```

Comment:
This should be `disabled` if the current user is not an admin? Or maybe we don't allow editing any users unless the user is an admin...

This also needs to be reflected in the access control of the user routes.